Destination/Destination.php

```diff
@@ -3,15 +3,25 @@
 namespace Zenstruck\BackupBundle\Destination;
 
 use Psr\Log\LoggerInterface;
+use \Countable;
+use \IteratorAggregate;
 
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-interface Destination
+interface Destination extends Countable, IteratorAggregate
```

Comment:
How to implement GUI over this if there is no way to iterate over the Destination? How to get list of current backups?

Sent from my iPhone

> On 11 Sep 2015, at 14:37, Kevin Bond <notifications@github.com> wrote:
> 
> In Destination/Destination.php:
> 
> >  
> >  /**
> >   * @author Kevin Bond <kevinbond@gmail.com>
> >   */
> > -interface Destination
> > +interface Destination extends Countable, IteratorAggregate
> I would like the destination interface to remain as is. There is no reason for S3CmdDestinaion to implement Countable, IteratorAggregate. If you want RotatingStreamDestination to implement them, that's fine.
> 
> —
> Reply to this email directly or view it on GitHub.
>